ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -228,6 +230,43 @@ type TangConfig struct {
 	Thumbprint string `yaml:"thumbprint" json:"thp"`
 }
 
+type DiskPartition struct {
+	Device     string       `yaml:"device"`
+	Partitions []Partitions `yaml:"partitions"`
+}
+
+type Partitions struct {
+	Label         string `yaml:"label"`
+	Size          int    `yaml:"size"`
+	Start         int    `yaml:"start"`
+	MountFileName string `yaml:"-"`
+	MountPath     string `yaml:"-"`
+}
+
+func (prt *Partitions) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	type PartitionsDefault Partitions
+	var defaults = PartitionsDefault{}
+	err := unmarshal(&defaults)
+
+	// https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-advanced_vardisk_installing-bare-metal
+	if !(strings.HasPrefix(defaults.Label, "var")) {
+		defaults.Label = "var-" + defaults.Label
```

Comment:
We should look at what Verizon is doing and see if this meets their needs.  If they're just creating partitions but not mounting them, do we need to enforce a leading "var-"?

Reply:
It's possible they may be making partitions that they're not mounting; depends a lot on the applications that I don't know anything about.

Reply:
Gotcha. Removed it.